components/supervisor/pkg/supervisor/tasks.go

```diff
@@ -516,6 +523,7 @@ func (tm *tasksManager) watch(task *task, term *terminal.Term) {
 
 		endMessage := "\r\n🍊 This task ran as a workspace prebuild\r\n" + duration + "\r\n"
 		_, _ = writer.Write([]byte(endMessage))
+		log.Error("Added kumquat line to prebuild log")
```

Comment:
Suggestion: Remove, or turn into log.Info

Reply:
Forgot about the .Error. Think it's fine to remove for now since we have the waitgroup making sure we're good with the timing.